Assets/_BForBoss/_Entities/Scripts/DummyTargetBehaviour.cs

```diff
@@ -59,7 +59,7 @@ public override void Reset()
             _healthbar.Reset();
         }
 
-        protected override void CleanUp()
+        public override void CleanUp()
```

Comment:
Nitpick: looks like `TriggerHitAnimation` and `LifeCycleFinished` are private and protected respectively and should below the public methods

Reply:
Fixed